core/templates/dev/head/pages/Base.js

```diff
@@ -41,5 +41,16 @@ oppia.controller('Base', [
       SidebarStatusService.onDocumentClick();
       $scope.$apply();
     });
+
+    $scope.skipTo = function() {
+      $scope.blockContent = document.getElementById('oppia-block-content');
+
+      if (!$scope.blockContent) {
+        console.log('no skip id');
```

Comment:
Throw an error here: `throw Error(...)`

Also, can you write an e2e test for this? It should press Tab once (or do whatever's needed to trigger the skip link), and then trigger said link. Then, you can check that, if the element with the desired ID doesn't exist on the page, an error actually does get thrown and the e2e test fails. This ensures that future changes don't break your code.